### ISSUE TITLE
Why did you remove the "generator"?

### ISSUE THREAD

**Issue by rsibanez89** [tjwebb/sails-auth#162]
I understand that using **marlinspike** you actually don't need to copy all the files to the root of the project.
Marlinspike exposes all that code that is in the "/node_modules/sails-auth", and "magically" we can create users and authenticate them.
My question is why? As a developer, I would actually prefer the code in the root of my project (copied by using the generator).
At least I know where each controller is and I have everything in the same place (Controllers, Models, Policies...). 
But probably is for maintainability. I think is a bit insecure if **marlinspike** is exposing a controller and I don't know about it. 

Thanks

**Comment by TroX-oO** [comment]
I totally agree with that. Even if it is not a default behaviour. I would prefer having the choice to get the code inside my project if I want to.
